src/functions/legacy/overrideSeverity.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+import { createRulesetFunction } from "@stoplight/spectral-core";
+//import { JsonPath } from '@stoplight/types';
```

Comment:
remove unused code and packages?

Reply:
This should be changed in https://github.com/ukhsa-collaboration/standards-api/commit/7c5b9f2641ff2259e44bf629dad29a9b6697bf18